README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
https://travis-ci.org/atom/language-puppet is already active.  I'll get back to you with Appveyor.